grammars/javascript.cson

```diff
@@ -633,6 +833,14 @@
   }
 ]
 'repository':
+  'operators':
```

Comment:
I did it to include operators in 'export' pattern:
```js
export default 123 + 23;
export var x = 0;
etc.
```
Is order important in 'repository' section?